dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLineInterfaceIIDM.h

```diff
@@ -52,6 +54,31 @@ class LineInterfaceIIDM : public LineInterface {
    */
   explicit LineInterfaceIIDM(powsybl::iidm::Line& line);
 
+  // Delete copy constructors and assignment operator because of destruction of IIDM extensions pointers in destructor
```

Comment:
We usually inherit from boost:non_copyable to do this

Reply:
OK